cmake/modules/SwiftSharedCMakeConfig.cmake

```diff
@@ -68,9 +68,9 @@ macro(swift_common_standalone_build_config_llvm product is_cross_compiling)
     set(${product}_NATIVE_LLVM_TOOLS_PATH "${LLVM_TOOLS_BINARY_DIR}")
   endif()
 
-  find_program(SWIFT_TABLEGEN_EXE "llvm-tblgen" "${${product}_NATIVE_LLVM_TOOLS_PATH}"
+  find_program(LLVM_TABLEGEN_EXE "llvm-tblgen" "${${product}_NATIVE_LLVM_TOOLS_PATH}"
```

Comment:
The problem with putting an assert in is how do you differentiate the value being set by LLVM's exported modules vs the value being explicitly set by a user?

In theory we could check before we include LLVMConfig to see if the value is set, and if so not trigger the assert... but I'm not sure that is really the behavior you want.

I'm also not really sure what value an assert would add. In the future if LLVM_TABLEGEN_EXE were exported by LLVMConfig, this line should just become a no-op because if the output variable is set the find doesn't execute. Of course the one difference between LLVMConfig and `find_program` is that `find_program` caches, where as LLVMConfig doesn't.